bftclient/include/bftclient/fake_comm.h

```diff
@@ -113,12 +113,14 @@ class FakeCommunication : public bft::communication::ICommunication {
   bool isRunning() const override { return true; }
   ConnectionStatus getCurrentConnectionStatus(NodeNum node) override { return ConnectionStatus::Connected; }
 
-  int send(NodeNum destNode, std::vector<uint8_t>&& msg) override {
+  int send(NodeNum destNode, std::vector<uint8_t>&& msg, NodeNum endpointNum) override {
     runner_.send(MsgFromClient{ReplicaId{(uint16_t)destNode}, std::move(msg)});
     return 0;
   }
 
-  std::set<NodeNum> send(std::set<NodeNum> dests, std::vector<uint8_t>&& msg) override {
+  std::set<NodeNum> send(std::set<NodeNum> dests,
+                         std::vector<uint8_t>&& msg,
+                         NodeNum endpointNum = MAX_NODE_NUM) override {
```

Comment:
I think it's enough to set a default in the base class.

Reply:
done